pkg/cmd/passholetemplatefuncs.go

```diff
@@ -0,0 +1,127 @@
+package cmd
+
+import (
+	"bytes"
+	"errors"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/mitchellh/go-homedir"
+
+	"github.com/twpayne/chezmoi/v2/pkg/chezmoi"
+	"github.com/twpayne/chezmoi/v2/pkg/chezmoilog"
+)
+
+type passholeConfig struct {
+	Command      string
+	Args         []string
+	Config       chezmoi.AbsPath
+	Database     chezmoi.AbsPath
+	Keyfile      chezmoi.AbsPath
+	NoPassword   bool
+	NoCache      bool
+	CacheTimeout int
+	cache        map[string]string
+	password     string
+}
+
+func raise(err error) {
```

Comment:
No other code in chezmoi uses a function like `raise`. When making PRs to projects it's important to follow the project's conventions.